doc/templates/command/template.cpp

```diff
@@ -22,31 +29,33 @@ void TemplateCommand::set_argument_parser() {
     //
     // Set a description that would be displayed in the second column of the
     // help message would also add the command to the parser
-    cmd.set_short_description("A command that prints its name and arguments' name");
+    cmd.set_description("A command that prints its name and arguments' name");
 
     // Add foo and bar options
     //
     // Option 1: as said in the header file here you should handle the pointer
     // by giving up the ownership to dnf5's parser.
     // Set the default value here.
     foo_option = dynamic_cast<libdnf5::OptionBool *>(
-        parser.add_init_value(std::unique_ptr<libdnf5::OptionBool>(new libdnf5::OptionBool(false))));
+        parser.add_init_value(
+            std::unique_ptr<libdnf5::OptionBool>(
+                new libdnf5::OptionBool(false))));
```

Comment:
@kontura My only problem with this (which the pre-commit formatter wants to change back to the un-wrapped format:)

```c++
    foo_option = dynamic_cast<libdnf5::OptionBool *>(
￼       parser.add_init_value(std::unique_ptr<libdnf5::OptionBool>(new libdnf5::OptionBool(false))));
```

...is that it won't fit when it's embedded on the documentation page, without forcing the reader to scroll horizontally.

Is there an override comment that can stop the formatter from re-wrapping it? I really think it reads better, for the documentation, if the line length is kept down to fit the page format.

Reply:
@kontura Done, thanks! I think that looks much better. I also made some other changes, I'll add some review comments about those.